twins/mongodb/states.go

```diff
@@ -39,6 +39,20 @@ func (sr *stateRepository) Save(ctx context.Context, st twins.State) error {
 	return nil
 }
 
+// Update persists the state
+func (sr *stateRepository) Update(ctx context.Context, st twins.State) error {
+	coll := sr.db.Collection(statesCollection)
+
+	filter := bson.D{{"id", st.ID}, {"twinid", st.TwinID}}
```

Comment:
It's normal `twinid` ?

Reply:
Yes. Automatically (un)marshalled (from)to `TwinID`.